src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonManager.cs

```diff
@@ -632,13 +634,11 @@ private ActorSelection Peer(Address at)
             return Context.ActorSelection(Self.Path.ToStringWithAddress(at));
         }
 
-        private void GetNextOldestChange()
+        private void GetNextOldestChanged()
         {
-            if (_oldestChangedReceived)
-            {
-                _oldestChangedReceived = false;
-                _oldestChangedBuffer.Tell(OldestChangedBuffer.GetNext.Instance);
-            }
+            if (!_oldestChangedReceived) return;
+            _oldestChangedReceived = false;
```

Comment:
So if I understand this correctly, this code can never run inside the `ClusterSingletonState.Start`, right?

Reply:
This code hasn't change, I just inverted the "if" statement.

Reply:
Got it